modules/persistent-storage-csi-dynamic-provisioning-aws-efs.adoc

```diff
@@ -22,8 +22,8 @@ Using monitoring of EFS volume sizes in AWS is strongly recommended.
 
 .Prerequisites
 
-* You have created AWS EFS volumes.
-* You have created the AWS EFS storage class.
+* You have created Amazon EFS volumes.
+* You have created the Amazon EFS storage class.
```

Comment:
Revoking seems correct for Line 25 and 26.

Reply:
As per the [comment](https://github.com/openshift/openshift-docs/pull/59650#discussion_r1290144425), updated Line accordingly and Revoking seems to be correct for Line 26.